src/detect-icmp-id.c

```diff
@@ -115,7 +111,7 @@ static inline bool GetIcmpId(Packet *p, uint16_t *id)
         return false;
     }
 
-    *id = pid;
+    *id = ntohs(pid);
```

Comment:
nit: SCNtohs

I see the codebase isn't fully clean, so I wonder if we still need it. It was added a long time ago to work around some windows issue.